common/types/cards.ts

```diff
@@ -49,17 +37,20 @@ export type HermitAttackT = {
 }
 
 export type CardDefs = {
-	type: CardTypeT
+	type?: CardTypeT
```

Comment:
Nice solution here to allow effect and single use defs to include the base card defs. That should be added here in the ts code.

Reply:
Actually this doesn't work so well, because each card does need to have a type - it can't be optional, even if it's not passed into the defs for different card types

Reply:
We can go back to using their own defs types instead of combining them.